### DISCUSSION TITLE
Multi-Tenancy Status and Roadmap

### DISCUSSION THREAD

**Discussion by laquereric** [marklogic-community/envision#257] [General]
What multi-tenancy features are offered now? Are there plans to expand these features? Should a concept of namespaces considered a part of this or should that be seperate discussion?

**Comment by wpaven** [comment]
Hi Eric,
There are no plans for multi-tenancy at this time.  The plan is for the visual elements of CE to be integrated into HC so multi-tenant requirements/requests should go to Product Management.  I'm unaware what the plan for multi-tenancy is in HC. 
Hope this helps,
Pete
